db/schema.rb

```diff
@@ -11,7 +11,7 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20161118012333) do
+ActiveRecord::Schema.define(version: 20161115154942) do
```

Comment:
Might want to watch out when staging files, this will cause conflicts with teammates for no real reason. Admittedly this file is one of my pain points with rails, because rake updates it even if there aren't changes 😦